### PR TITLE
Add artisan command `lighthouse:cache` to compile GraphQL AST

### PR DESCRIPTION
**Pull request by tlaverdure**
- [x] Added or updated tests
- [X] Documented user facing changes
- [X] Updated CHANGELOG.md

**Changes**
Adds a command to create the AST cache. Currently need this to prime the cache before deployment to a serverless environment. Ideally I'd like to embed the cache with the deployment artifact.

**Breaking changes**

N/A

### CODE REVIEW

**Comment by spawnia** [comment]
Seems useful. The name is probably fine, since it aligns with built-in Laravel commands.

I feel like the descriptions should contain the word *compile* (sounds cool).

**Comment by tlaverdure** [comment]
Haha, sure I'll revise.

File: docs/master/api-reference/commands.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 Lighthouse provides some convenient artisan commands. All of them
 are namespaced under `lighthouse`.
 
+## cache
+
+Complie the GraphQL AST cache.
```
Comment (spawnia) [inline docs/master/api-reference/commands.md:8]:
```suggestion
Compile the GraphQL AST cache.
```

The actual command description should match.

**Comment by tlaverdure** [comment]
No idea why this test is failing for that specific version.

**Comment by spawnia** [comment]
> No idea why this test is failing for that specific version.

Might be because of the Facade usage, we try not to use them anyways.

**Comment by tlaverdure** [comment]
Ok will switch to a helper.

**Comment by spawnia** [comment]
Hm, i am guessing that the cache is somehow lost between calls. You can try and install an older Laravel version locally for debugging.

    rm composer.lock && composer require illuminate/contracts:5.7

**Comment by tlaverdure** [comment]
Looks like `Cache::remember()` needs a ttl in that version. Good to go 👍🏾

**Pull request merged**